node/src/BaseClient.ts

```diff
@@ -3438,6 +3439,42 @@ export class BaseClient {
         return this.createWritePromise(createZScore(key, member));
     }
 
+    /**
+     * Computes the union of sorted sets given by the specified `keys` and stores the result in `destination`.
+     * If `destination` already exists, it is overwritten. Otherwise, a new sorted set will be created.
+     * To get the result directly, see {@link zunionWithScores}.
+     *
+     * @see {@link https://valkey.io/commands/zunionstore/|valkey.io} for details.
+     * @remarks When in cluster mode, all keys in `keys` must map to the same hash slot.
+     * @param destination - The key of the destination sorted set.
+     * @param keys - The keys of the sorted sets with possible formats:
+     *         string[] - for keys only.
```

Comment:
```suggestion
     * - string[] - for keys only.
```
Otherwise you don't get the indentation: 
![image](https://github.com/user-attachments/assets/ed4f9290-4645-44cf-b6c7-39608d7a28ab)